configs/projects/bolt-runtime.rb

```diff
@@ -178,7 +179,7 @@
   proj.component 'rubygem-terminal-table'
   proj.component 'rubygem-unicode-display_width'
 
-  # Core Windows dependencies
+  # Core Windows dependenciesq
```

Comment:
nit typo

Reply:
Yeah it looks like the testing linked above was done against https://github.com/puppetlabs/puppet-runtime/commit/4637dcdbf25b16ab97728be4d03873247fd83903 , then PR author realized that that commit duplicated listing ffi as a dependency in bolt-rutnime (where the trivial typo was introduced). I'm currently building a new batch of bolt-runtime to validate against bolt-vanagon.

Reply:
Built bolt-runtime off latest commit. Runtime artifacts: builds.delivery.puppetlabs.net/puppet-runtime/444fed62ab170c6a3e7a7a9db077497971389a87/artifacts/

Using those runtimes I build bolt. Bolt packages with updated gems: http://builds.delivery.puppetlabs.net/puppet-bolt/963adf3253685416667f3d766588c84a3131184f/artifacts/

No problems with https://jenkins-master-prod-1.delivery.puppetlabs.net/view/bolt/job/platform_bolt-vanagon_vanagon-integration-suite_no-PEZ-daily-master/591/

So I'm good with the content from the bolt-runtime side.